Cart-Page/cart-Responsive.css

```diff
@@ -0,0 +1,35 @@
+@media screen and (max-width: 975px) {
+    .card-product,.cart__detail{
+      margin: 1em 2em;
+    }
+  }
+  @media screen and (max-width: 650px) {
+    .cart__content{
+      grid-template-columns: 1fr;
+    }
+    .cart__detail{
+      order:1;
+    }
+    .card-product{
+      order:2;
+    }
+  }
+  @media screen and (max-width: 550px) {
```

Comment:
Is it really necessary to have one media query at 650px and 550px.

Reply:
No, ThankYou I have changed the value.